src/npm.py

```diff
@@ -0,0 +1,73 @@
+from glob import glob
+import os
+import re
+import sys
+import json
+import frontmatter
+import urllib.request
+from datetime import datetime
+from html.parser import HTMLParser
+from liquid import Template
+
+
+DEFAULT_TAG_TEMPLATE = (
+    "{{major}}{% if minor %}.{{minor}}{% if patch %}.{{patch}}{%endif%}{%endif%}"
+)
+REGEX = r"^(?:(\d+\.(?:\d+\.)*\d+))$"
+
+def fetch_releases(npm_id, regex):
+    releases = {}
+
+    if not isinstance(regex, list):
+        regex = [regex]
+
+    url = "https://registry.npmjs.org/%s" % npm_id
```

Comment:
We could use f-strings:
```suggestion
    url = f"https://registry.npmjs.org/{npm_id}"
```